WooCommerce/Classes/ViewRelated/Orders/Order Creation/FlowCoordinator/AddOrderCoordinator.swift

```diff
@@ -68,6 +73,23 @@ private extension AddOrderCoordinator {
         SimplePaymentsAmountFlowOpener.openSimplePaymentsAmountFlow(from: navigationController, siteID: siteID)
     }
 
+    /// Presents `BottomAnnouncementView`UIHostingController  modally.
+    ///
+    func presentBottomAnnouncement() {
+        let baseViewController = BaseViewController()
+        baseViewController.completionHandler = redirectToHubMenu
```

Comment:
As talked this can be added into a View Model

Reply:
Done at 94997e002f6e4f4c90a6260b45fa98f8f4d30157 👍